src/amp.js

```diff
@@ -43,6 +44,7 @@ try {
     try {
```

Comment:
Install performanceFor before installStyles and tick there and at the end of this callback as our first 2 ticks.